src/Symfony/Component/Yaml/Inline.php

```diff
@@ -28,6 +28,7 @@ class Inline
     private static $exceptionOnInvalidType = false;
     private static $objectSupport = false;
     private static $objectForMap = false;
+    private static $constantSupport = false;
```

Comment:
I agree with that. It will consistent with how we already treat object PHP support.